README.md

```diff
@@ -4,6 +4,23 @@
 
 ## Guide
 
+### GitHub Actions
+A minimal configuration for public repos:
+
+```yml
+steps:
+  # (Other steps go here)
+  - name: "Upload coverage to Codecov"
+    uses: codecov/codecov-action@v1
+    with:
+      fail_ci_if_error: true
+      verbose: true
```

Comment:
```suggestion
```

Going to remove `verbose` because it's not necessary

Reply:
Fine by me! Thanks for the review.